packages/orbit-components/src/primitives/MobileDialogPrimitive/components/DialogContent.flow.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import type { Globals } from "../../../common/common.js.flow";
+
+export type Props = {|
+  shown: boolean,
+  dialogId: ?string,
+  children: React$Node,
+  onClose: () => void,
+  ...Globals,
+|};
+
+declare export default React.ComponentType<Props>;
```

Comment:
We're using `React.ComponentType` but it's not imported. Also we should prohibit usage of `React$Node`.